docker/Dockerfile

```diff
@@ -20,18 +24,20 @@ COPY cheribuild.json /root/.config/cheribuild.json
 
 # deps to build QEMU+elftoolchain:
 RUN apt-get update && apt-get install -y \
-  libtool pkg-config python-minimal autotools-dev automake autoconf libglib2.0-dev libpixman-1-dev \
+  libtool pkg-config python2-minimal autotools-dev automake autoconf libglib2.0-dev libpixman-1-dev \
   bison groff-base libarchive-dev flex
 
-# INSTALL clang 8.0
+# INSTALL clang 10.0
 RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
 RUN echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main" > /etc/apt/sources.list.d/llvm.list
 # RUN echo "deb http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu trusty main" > /etc/apt/sources.list.d/r-toolchain.list
 # RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 1E9377A2BA9EF27F
-RUN apt-get update && apt-get install -y clang-8 lld-8
+RUN apt-get update && apt-get install -y clang-10 lld-10
 
 RUN apt-get update && apt-get install -y cmake
 
+RUN groupadd -g 12345 -r cheri && useradd -u 12345 --no-log-init -r -g cheri cheri
```

Comment:
No home directory? Also, adduser is preferred and can create both the group and user in one command